scripts/ci-build-azure-ccm.sh

```diff
@@ -46,12 +49,12 @@ setup() {
 }
 
 main() {
-    if [[ "$(can_reuse_artifacts)" == "false" ]]; then
+    if [[ "$(can_reuse_artifacts)" =~ "false" ]]; then
```

Comment:
When CCM image exists and CNM image doesn't exist, $(can_reuse_artifacts) will return something like "<Image_tag>: Pulling from......false".